checks/481552.yaml

```diff
@@ -0,0 +1,83 @@
+id: "481552"
+name: hana_call_timeout
+group: Pacemaker
+description: |
+  HANA_CALL_TIMEOUT has correct value
+remediation: |
+  ## Abstract
+  HANA_CALL_TIMEOUT should have recommended value in cluster configuration. No configuration is also allowed as it will then use the default value.
+
+  ## Remediation
+  Edit the crm configuration and change the value for this parameter to 60.
+
+  ## References
+  AZURE:
+
+    - https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/high-availability-guide-suse-pacemaker
+
+  AWS:
+
+    - https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/
+
+  GCP:
+
+    - https://cloud.google.com/solutions/sap/docs/sap-hana-ha-config-sles
+
+  SUSE / KVM:
+
+    - https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/
+
+  Nutanix:
+
+    - https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/
+
+severity: warning
+metadata:
+  target_type: cluster
+  cluster_type: 
+    - hana_scale_up
+    - hana_scale_out
+  architecture_type: 
+    - angi
+    - classic
+
+facts:
+  - name: resources
+    gatherer: cibadmin@v1
+    argument: cib.configuration.resources
+
+values:
+  - name: expected_hana_call_timeout
+    default: 60      
+
+expectations:
+  - name: expectations_hana_call
+    expect: |
+        let resources = facts.resources;
+        let masters = [];
+        let clones = [];
+
+        if "master" in resources {
+          masters = resources.master;
+        }
+        if "clone" in resources {
+          clones = resources.clone;
+        }
+
+
+
+        let masters_and_clones = masters + clones;
+        if is_empty(masters_and_clones) {
+          return false;
+        }
+
+        for resource_group in masters_and_clones {
+          for primitive in resource_group.primitive {
+            if primitive.type == "SAPHana" || primitive.type == "SAPHanaController" {
+              let attr = primitive.instance_attributes.nvpair.find(|nvpair| nvpair.name == "HANA_CALL_TIMEOUT");
+              if attr == () || attr.value == 60 {return true;}
```

Comment:
Shouldn't the `60` be `values.expected_hana_call_timeout` ?

Reply:
Yes. I will change it